packages/altive_lints/lib/src/lints/avoid_single_child.dart

```diff
@@ -85,4 +96,12 @@ class AvoidSingleChild extends DartLintRule {
       }
     });
   }
+
+  bool _hasMultipleChild(CollectionElement element) {
+    if (element is SpreadElement && element.expression is ListLiteral) {
+      final spreadElement = element.expression as ListLiteral;
+      return spreadElement.elements.length != 1;
```

Comment:
![badge](https://img.shields.io/badge/review-ask-informational)
@boywithdv 
Is there a problem if `isEmpty` is true, as it seems to `return true` as well?
```suggestion
      return spreadElement.elements.length > 1;
```

Reply:
It seems more natural to set it to 1 or more, since it returns true even if it is empty.

I was not able to assume that the number of elements in the spread operator is empty.

We will also create a test case for the case where the number of elements of the spread operator is empty.
Thank you very much.